controller/trackerquery.go

```diff
@@ -138,11 +155,22 @@ func (c *TrackerqueryController) Delete(ctx *app.DeleteTrackerqueryContext) erro
 	if err != nil {
 		return jsonapi.JSONErrorResponse(ctx, goa.ErrUnauthorized(err.Error()))
 	}
+
 	err = application.Transactional(c.db, func(appl application.Application) error {
 		tq, err := appl.TrackerQueries().Load(ctx.Context, ctx.ID)
 		if err != nil {
 			return errs.Wrapf(err, "failed to delete tracker query %s", ctx.ID)
 		}
+
+		// check if user is space collaborator
+		authorized, err := authz.Authorize(ctx, tq.SpaceID.String())
```

Comment:
Could we look for the `contribute` scope here as well?

Reply:
@sbose78 addressed it here 4085595